src/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import rootReducer from './reducers';
+
+const middleware = [thunk];
```

Comment:
**[comment]** This is a bit of a paucity of middleware, no? Some problems worth solving:

- logging actions
- enforcing the immutability of state in components
- batching related actions
- form handling
- and much more